deploy/helm/flightctl/templates/flightctl-kv-statefulset.yaml

```diff
@@ -55,7 +55,6 @@ spec:
         flightctl.service: flightctl-kv
     spec:
       securityContext:
```

Comment:
empty `securityContext` should be removed too

Reply:
Done